umap/static/umap/map.css

```diff
@@ -1415,10 +1415,10 @@ a.add-datalayer:hover,
 .leaflet-contextmenu-icon {
     display: none;
 }
-.umap-popup-large iframe,
-.umap-popup-large img {
+.umap-popup-large iframe {
     /* See https://github.com/Leaflet/Leaflet/commit/61d746818b99d362108545c151a27f09d60960ee#commitcomment-6061847 */
-    max-width: 500px !important;
+    max-width: 100% !important;
+    min-width: 500px;
```

Comment:
I guess it will break on small devices, did you try?

Reply:
"Fixed"!
Sadly, I think popup sizing still needs a refacto, specifically for small screens but not only, mainly because it's partly done in JS and partly in CSS, and mainly because of [this](https://github.com/Leaflet/Leaflet/commit/61d746818b99d362108545c151a27f09d60960ee#commitcomment-6061847)